src/SFA.DAS.EmployerAccounts.Web.UnitTests/Controllers/EmployerTeamControllerTests/WhenHasSingleDraftVacancy.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Moq;
+using NUnit.Framework;
+using SFA.DAS.Authentication;
+using SFA.DAS.Authorization;
+using SFA.DAS.EAS.Portal.Client;
+using SFA.DAS.EmployerAccounts.Interfaces;
+using SFA.DAS.EmployerAccounts.Web.Controllers;
+using SFA.DAS.EmployerAccounts.Web.Orchestrators;
+using SFA.DAS.EmployerAccounts.Web.ViewModels;
+using SFA.DAS.HashingService;
+using System.Web.Mvc;
+using Model = SFA.DAS.EAS.Portal.Client.Types;
+
+namespace SFA.DAS.EmployerAccounts.Web.UnitTests.Controllers.EmployerTeamControllerTests
+{
+    public class WhenHasSingleDraftVacancy
+    {
+        private EmployerTeamController _controller;
+
+        private Mock<IAuthenticationService> mockAuthenticationService;
+        private Mock<IAuthorizationService> mockAuthorizationService;
+        private Mock<IMultiVariantTestingService> mockMultiVariantTestingService;
+        private Mock<ICookieStorageService<FlashMessageViewModel>> mockCookieStorageService;
+        private Mock<EmployerTeamOrchestrator> mockEmployerTeamOrchestrator;
+        private Mock<IPortalClient> mockPortalClient;
+        private Mock<IHashingService> mockHashingService;
+
+        [SetUp]
+        public void Arrange()
+        {
+            mockAuthenticationService = new Mock<IAuthenticationService>();
+            mockAuthorizationService = new Mock<IAuthorizationService>();
+            mockMultiVariantTestingService = new Mock<IMultiVariantTestingService>();
+            mockCookieStorageService = new Mock<ICookieStorageService<FlashMessageViewModel>>();
+            mockEmployerTeamOrchestrator = new Mock<EmployerTeamOrchestrator>();
+            mockPortalClient = new Mock<IPortalClient>();
+            mockHashingService = new Mock<IHashingService>();
+
+            _controller = new EmployerTeamController(
+                mockAuthenticationService.Object,
+                mockAuthorizationService.Object,
+                mockMultiVariantTestingService.Object,
+                mockCookieStorageService.Object,
+                mockEmployerTeamOrchestrator.Object,
+                mockPortalClient.Object,
+                mockHashingService.Object);
+        }
+
+        [Test]
+        public void ThenTheVacancyStatusViewIsReturned()
+        {
+            // Arrange
+            var model = new AccountDashboardViewModel();
+            model.PayeSchemeCount = 1;
+            model.AgreementsToSign = false;
+
+            model.AccountViewModel = new Model.Account();
+            model.AccountViewModel.Vacancies.Add(new Model.Vacancy { Status = Model.VacancyStatus.Draft });
```

Comment:
The `Account` model has changed in the latest portal client. It no longer has a collection of vacancies.